src/pages/index.js

```diff
@@ -21,6 +23,12 @@ const lastSection = css`
   margin-bottom: 30rem;
 `
 
+const subscribeSection = css`
+  margin: 0 auto;
+  width: 102.4rem;
+  padding: 0 1.2rem;
+`
```

Comment:
Add following to match design:
```suggestion
box-sizing: border-box;
`
```

Reply:
added.